starknet_py/net/gateway_client.py

```diff
@@ -96,7 +98,7 @@ async def get_block(
         res = await self._feeder_gateway_client.call(
             method_name="get_block", params=block_identifier
         )
-        return StarknetBlockSchema().load(res, unknown=EXCLUDE)
+        return StarknetBlockSchema().load(res, unknown=EXCLUDE)  # pyright: ignore
```

Comment:
Why are these ignores needed? Perhaps something like 
```py
wrapped_res: StarknetBlock = StarknetBlockSchema().load(res, unknown=EXCLUDE)
return wrapped_res
```
or a usage of `typing.cast` would be a better solution?

Reply:
It's more code, I am ok with having ignores here.

Reply:
This doesn't seem ideal, but i guess we could keep it. However `typing.cast` doesn't add that much code imo.